index.js

```diff
@@ -1,55 +1,58 @@
 const Response = {
 
-    success(body) {
-        return this.buildResponse(200,body)
-    },
-
-    resourceCreated(body) {
-        return this.buildResponse(201, body)
-    },
-
-    error(body) {
-        return this.buildResponse(500,body)
-    },
-
-    notFound(body) {
-        return this.buildResponse(404,body)
-    },
-
-    badRequest(body) {
-        return this.buildResponse(400,body)
-    },
-
-    alreadyExists(body) {
-        return this.buildResponse(409,body)
-    },
-
-    validationError(body) {
-        return this.buildResponse(422,body)
-    },
-
-    buildResponse(statusCode,body) {
-        if (typeof body === 'object') {
-            body = JSON.stringify(body)
-        }
-
-        return {
-            statusCode,
-            body,
-        }
+  success (body) {
+    return this.buildResponse(200, body)
+  },
+
+  resourceCreated (body) {
+    return this.buildResponse(201, body)
+  },
+
+  error (body) {
+    return this.buildResponse(500, body)
+  },
+
+  notFound (body) {
+    return this.buildResponse(404, body)
+  },
+
+  badRequest (body) {
+    return this.buildResponse(400, body)
+  },
+
+  unauthorized (body) {
+    return this.buildResponse(401, body)
+  },
+
+  alreadyExists (body) {
+    return this.buildResponse(409, body)
+  },
+
+  validationError (body) {
+    return this.buildResponse(422, body)
+  },
+
+  buildResponse (statusCode = 200, body) {
+    if (typeof body === 'object') {
+      body = JSON.stringify(body)
     }
+
+    return {
+      statusCode: parseInt(statusCode),
+      body
+    }
+  }
 }
 
 module.exports = {
-    Response,
-    Errors: {
-        Unauthorized: require('./module/errors/Unauthorized'),
-        UnprocessableEntity: require('./module/errors/UnprocessableEntity'),
-        InternalServerError: require('./module/errors/InternalServerError'),
-        Forbidden: require('./module/errors/Forbidden'),
-        NotFound: require('./module/errors/NotFound'),
-        Fail: require('./module/errors/Fail'),
-        AlreadyExists: require('./module/errors/AlreadyExists')
-    },
-
-}
\ No newline at end of file
+  Response,
+  Errors: {
+    Unauthorized: require('./module/errors/Unauthorized'),
+    UnprocessableEntity: require('./module/errors/UnprocessableEntity'),
+    InternalServerError: require('./module/errors/InternalServerError'),
+    Forbidden: require('./module/errors/Forbidden'),
+    NotFound: require('./module/errors/NotFound'),
+    Fail: require('./module/errors/Fail'),
+    AlreadyExists: require('./module/errors/AlreadyExists')
```

Comment:
This pattern is super interesting! Python's style guide explicitly disallows `import`ing anywhere other than at the top of a file.  But I think this is a novel way to present resources to a caller without having to pretend like I have any interest other than just forwarding those resources.

Reply:
I agree @rsc-rfrench , I love this pattern. I've used it for express routing too so I could have my route documentation live in a single file, but have the actual implementation of the route live in another.